gemfiles/rails_3.gemfile

```diff
@@ -11,7 +11,7 @@ gem "dotenv", :require => false
 gem "simplecov", :require => false
 gem "codecov", :require => false
 gem "resque", :require => false
-gem "sidekiq", :require => false
+gem "sidekiq", '~> 5.2.10', :require => false
```

Comment:
Sidekiq 5 and Ruby 3 [aren't compatible](https://github.com/mperham/sidekiq/blob/main/docs/5.0-Upgrade.md#whats-new), not sure if it's so relevant here

Reply:
updated